test/compose/docker-compose.yml

```diff
@@ -10,13 +10,23 @@ services:
       MINIO_SECRET_KEY: GMusLtUmILge2by+z890kQ
     entrypoint: /bin/ash -c 'mkdir -p /data/backup && minio server /data'
     volumes:
-      - backup_data:/data
+      - minio_backup_data:/data
+
+  webdav:
+    image: bytemark/webdav:2.4
+    environment:
+      AUTH_TYPE: Digest
+      USERNAME: test
+      PASSWORD: test
+    volumes:
+      - webdav_backup_data:/data
```

Comment:
Looking at this https://hub.docker.com/r/bytemark/webdav/ I think you will need to mount the volume at `/var/lib/dav` instead of `/data` as this is where the server stores its data. Files will then appear at `/var/lib/dav/data`